### DISCUSSION TITLE
Boa tarde! Preciso resolver esse meu problema no Flutter que não consigo de jeito nenhum. Alguem poderia me ajuda ?

### DISCUSSION THREAD

**Discussion by luizfelipe02081** [Flutterando/forum#124] [Perguntas e Respostas]
Flutter doctor.

![Captura de Tela (50)](https://user-images.githubusercontent.com/119639601/221423610-a5abdd27-335b-46bc-9c28-60cce53b6be7.png)

JAVA HOME.

![Captura de Tela (51)](https://user-images.githubusercontent.com/119639601/221423650-ce7c4b1b-2a2f-4556-8da9-7564f3a73d42.png)

**Comment by EuHigorBarbosa** [comment]
Você deu muito pouca informação sobre seu problema...não disse o que você deseja e nem o que você já tentou. Você precisa gastar um tempo, dedicar tempo na elaboração de sua pergunta pois assim, quem desejar te ajudar vai poder ajudar. 

Do jeito que você fez me pareceu sem direção. Por exemplo, se seu flutter doctor deu essas issues, que tal ir resolvendo uma a uma?

**Comment by rdsimas** [comment]
A variável JAVA_HOME não deve apontar pro instalável .exe, mas sim para a pasta onde está a JDK.

O padrão é assim: C:\Program Files\Java\jdk-11.0.7
